### PR TITLE
[WR-253] Sync mfa settings api

### PR DESCRIPTION
**Pull request by Mrhea**
Committing this so I can update identity service with a new version

### CODE REVIEW

File: identityClient/api.go [hunk 1]
```diff
@@ -1120,6 +1120,19 @@ type GetFederatedIdentitiesForSyncRequest struct {
 	PrimaryRealmEndpoint string
 }
 
+// TotpMFASettings wraps the information about an Identity's MFA settings, if any.
+type TotpMFASettings struct {
+	ID             string `sql:"id"`
```
Comment (galxy25) [inline identityClient/api.go:1125]: `json:"id"` here and elsewhere

**Review by galxy25 — APPROVED** [review]

File: identityClient/api.go (hunk 1 above)
Comment (Mrhea) [reply identityClient/api.go:1125]: Oops! I was working in identity's db so I didn't context switch 😓

File: identityClient/api.go [hunk 2]
```diff
@@ -1120,6 +1120,19 @@ type GetFederatedIdentitiesForSyncRequest struct {
 	PrimaryRealmEndpoint string
 }
 
+// TotpMFASettings wraps the information about an Identity's MFA settings, if any.
+type TotpMFASettings struct {
+	ID             string `json:"id"`
+	Salt           []byte `json:"salt"`
+	CredentialType string `json:"type"`
+	UserID         string `json:"user_id"`
```
Comment (rosasck) [inline identityClient/api.go:1128]: PrimaryUserID

**Review by rosasck — APPROVED** [review]
pending language change

**Pull request merged**